src/main/java/dev/openfeature/sdk/providers/memory/InMemoryProvider.java

```diff
@@ -47,88 +29,81 @@ public InMemoryProvider(Map<String, Flag<?>> flags) {
         this.flags = new ConcurrentHashMap<>(flags);
     }
 
-    /**
-     * Initializes the provider.
-     * @param evaluationContext evaluation context
-     * @throws Exception on error
-     */
-    @Override
-    public void initialize(EvaluationContext evaluationContext) throws Exception {
-        super.initialize(evaluationContext);
-        state = ProviderState.READY;
-        log.debug("finished initializing provider, state: {}", state);
-    }
-
```

Comment:
As a proof that nothing is breaking, you should be able to get away with making no changes to this provider.